### PR TITLE
added texts to success-pages

### PR DESCRIPTION
**Pull request by gorzala**

### CODE REVIEW

File: src/app/features/home/pages/contributor-registration-success/contributor-registration-success.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
+<p>Du willst mitmachen. Wir freuen uns auf dich!</p>
 
 <div class="space-container"></div>
-<h2>Was demnächst?</h2>
-<p>Du kannst uns gerne kontaktieren, wenn du irgendwelche Vorschläge oder Feedbacks zu uns hast.</p>
-<p>Und sobald unsere Webseite online ist, wirst du per Mail informiert. </p>
+<h2>Wie geht es weiter?</h2>
+<p>
+    In den nächsten Tagen wird sich jemand von uns bei Dir melden und
+     mit Dir besprechen wie du dich einbringen kannst.
```
Comment (jans510) [inline src/app/features/home/pages/contributor-registration-success/contributor-registration-success.component.html:8]: dir (klein)

File: src/app/features/home/pages/contributor-registration-success/contributor-registration-success.component.html [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+<h2>Viele Dank!</h2>
+<p>Du willst mitmachen. Wir freuen uns auf dich!</p>
 
 <div class="space-container"></div>
-<h2>Was demnächst?</h2>
-<p>Du kannst uns gerne kontaktieren, wenn du irgendwelche Vorschläge oder Feedbacks zu uns hast.</p>
-<p>Und sobald unsere Webseite online ist, wirst du per Mail informiert. </p>
+<h2>Wie geht es weiter?</h2>
+<p>
+    In den nächsten Tagen wird sich jemand von uns bei Dir melden und
```
Comment (jans510) [inline src/app/features/home/pages/contributor-registration-success/contributor-registration-success.component.html:7]: dir (klein)

Comment (gorzala) [reply src/app/features/home/pages/contributor-registration-success/contributor-registration-success.component.html]: done

File: src/app/features/home/pages/contributor-registration-success/contributor-registration-success.component.html (hunk 1 above)
Comment (gorzala) [reply src/app/features/home/pages/contributor-registration-success/contributor-registration-success.component.html]: done

**Review by manuela2013 — APPROVED** [review]

**Pull request merged**